recipes/holoviews-1.5.0/meta.yaml

```diff
@@ -0,0 +1,41 @@
+package:
+  name: holoviews
+  version: 1.5.0
+
+source:
+  path: .
```

Comment:
Thanks. What is the md5 sum of exactly? The archive generated after a conda build?